tests/test_utils.py

```diff
@@ -191,6 +191,19 @@ def get_dataset(dataset_type, data, schemas=None, autoinspect_func=autoinspect.c
     else:
         raise ValueError("Unknown dataset_type " + str(dataset_type))
 
+def candidate_getter_is_on_temporary_notimplemented_list(context, getter):
+    if context == 'SqlAlchemyDataset':
+        return getter in [
+            'get_column_count_in_range',
```

Comment:
@cselig : I did implement get_column_count_in_range for sqlalchemy (to use the spark release as an example of unlocking additional distributional expectations in sqlalchemy); could you remove that from the list?